server/src/middlewares/auth.js

```diff
@@ -13,28 +13,63 @@ const checkJwt = (req, res, next, prisma) => {
     )
   }
 
-  jwt({
-    secret: jwksRsa.expressJwtSecret({
-      cache: true,
-      rateLimit: true,
-      jwksRequestsPerMinute: 1,
-      jwksUri: `https://${conf.auth0Domain}/.well-known/jwks.json`
-    }),
-    credentialsRequired: true,
-    audience: conf.auth0ClientId,
-    issuer: `https://${conf.auth0Domain}/`,
-    algorithms: ['RS256']
-  })(req, res, next)
+  const authType =
+    req.headers.authorization && req.headers.authorization.split(' ')[0]
+
+  req.authType = authType
+
+  if (authType === 'Bearer') {
+    jwt({
+      secret: jwksRsa.expressJwtSecret({
+        cache: true,
+        rateLimit: true,
+        jwksRequestsPerMinute: 1,
+        jwksUri: `https://${conf.auth0Domain}/.well-known/jwks.json`
+      }),
+      credentialsRequired: true,
+      audience: conf.auth0ClientId,
+      issuer: `https://${conf.auth0Domain}/`,
+      algorithms: ['RS256']
+    })(req, res, next)
```

Comment:
So I'm guessing the middleware returned by `jwt(...)` checks the token present in the `Authorization` header? Is that right?

Reply:
Yes indeed